src/OSPSuite.Presentation/Presenters/Charts/CurveMultiItemEditorPresenter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.Views.Charts;
+using OSPSuite.Utility;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public interface ICurveMultiItemEditorPresenter : IDisposablePresenter, IPresenter<ICurveMultiItemEditorView>
+   {
+      SelectedCurveValues GetSelectedValues();
+      IEnumerable<bool?> AllBooleanOptions { get; }
+      IEnumerable<LineStyles?> AllLineStyles { get; }
+      IEnumerable<Symbols?> AllSymbols { get; }
+      void SetColorChangedFlag();
+   }
+
+   public class SelectedCurveValues
+   {
+      public Color Color { get; set; }
+      public LineStyles? Style { get; set; }
+      public Symbols? Symbol { get; set; }
+      public bool? Visible { get; set; }
+      public bool? VisibleInLegend { get; set; }
+      public bool IsColorSet { get; set; } = false;
+   }
+
+   public class CurveMultiItemEditorPresenter : AbstractDisposablePresenter<ICurveMultiItemEditorView, ICurveMultiItemEditorPresenter>,
+      ICurveMultiItemEditorPresenter
+   {
+      private readonly SelectedCurveValues _selectedCurveValues;
+
+      public CurveMultiItemEditorPresenter(ICurveMultiItemEditorView view) : base(view)
+      {
+         _selectedCurveValues = new SelectedCurveValues();
+      }
+
+      public IEnumerable<bool?> AllBooleanOptions { get; } = Constants.MultiCurveOptions.AllBooleanOptions;
+
+      public IEnumerable<LineStyles?> AllLineStyles { get; } = new List<LineStyles?>() {null}.Union(EnumHelper.AllValuesFor<LineStyles>().Cast<LineStyles?>());
+
+      public IEnumerable<Symbols?> AllSymbols { get; } = new List<Symbols?>() {null}.Union(EnumHelper.AllValuesFor<Symbols>().Cast<Symbols?>());
+
+      public void SetColorChangedFlag()
+      {
+         _selectedCurveValues.IsColorSet = true;
```

Comment:
I don't get this IsColorSet property. Why not Color?

Reply:
so the Color? did not work with binding to the ColorEdit Control. Also not with the formatter, so the best workaround I could think of was to use the flag.